snuba/datasets/transactions_processor.py

```diff
@@ -71,6 +76,9 @@ def process_message(self, message, metadata=None) -> Optional[ProcessedMessage]:
             processed["start_ts"], processed["start_ms"] = self.__extract_timestamp(
                 data["start_timestamp"],
             )
+            if data["timestamp"] - data["start_timestamp"] < 0:
+                # Seems we have some negative durations in the DB
+                metrics.increment('negative_duration')
```

Comment:
Seems like it'd be more appropriate to log this and include the message stream and offset so that we can look further?

Reply:
I will add a log message there after seeing how common this is. I suspect it could be fairly substantial, thus logging everyone of them may be too much.